cmd/ak/cmd/triggers/create.go

```diff
@@ -13,10 +13,11 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var event, filter, loc, name string
+var event, filter, loc, name, schedule string
 
 var createCmd = common.StandardCommand(&cobra.Command{
-	Use:     "create <--name=...> <--env=...> <--connection=...> <--event=...> <--loc=...>",
+	Use: `create <--name=...> <--env=...> <--connection=...> <--event=...> <--loc=...>
+                 <--name=...> <--env=...> <--schedule=...> <--loc=...>`,
```

Comment:
`Use` should be a single line. Also, I think we should remove `--env` but that's beyond the scope of this PR.

```suggestion
	Use: "create --name <...> --env <...> {--connection <...> --event <...> | --schedule <...>} --loc <...>",
```

Reply:
I could convert Use to single line, but it's less readable IMO. Why not multiline?

Reply:
See Cobra's documentation: "Use is the one-line usage message..."

https://pkg.go.dev/github.com/spf13/cobra#Command.Use